src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -483,4 +485,49 @@ public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
     }
 
+    public static String removeVariables(String statusMessage) {
```

Comment:
Some status messages in `Const.java` have templates in them: `You are enrolled in the course %s`. This is the message stored in the HashMap. The actual status message has the course ID in place of `%s`. In order to search for the message in the HashMap, I called the method for the template message and the actual message to remove `%s` and the actual course ID respectively.